src/consensus/tx_verify.h

```diff
@@ -56,12 +56,13 @@ unsigned int GetP2SHSigOpCount(const CTransaction& tx, const std::span<T> coins)
 
 /**
  * Compute total signature operation cost of a transaction.
- * @param[in] tx     Transaction for which we are computing the cost
- * @param[in] inputs Map of previous transactions that have outputs we're spending
+ * @param[in] tx    Transaction for which we are computing the cost
+ * @param[in] coins Sorted span of Coins containing previous transaction outputs we're spending
```

Comment:
In "consensus: Use Coin span in GetP2SHSigOpCount" 5709d07

Here and other commits.
Sorted in what order?